FetchError: request to https://sheets.googleapis.com/v4/spreadsheets/1vkw_Lku7F_F3F_iNmFFrDq9j7-tQ6EmZPOLpLt-s3TY/values/Cruise%20Sum%20By%20Day?key=AIzaSyDtI69z7dF2482nw6ZZbFaniKLDLMyFeIA failed, reason: connect ETIMEDOUT 172.217.15.74:443
    at ClientRequest.<anonymous> (/home/runner/work/covid19japan-data/covid19japan-data/node_modules/node-fetch/lib/index.js:1455:11)
    at ClientRequest.emit (events.js:315:20)
    at TLSSocket.socketErrorListener (_http_client.js:426:9)
    at TLSSocket.emit (events.js:315:20)
    at emitErrorNT (internal/streams/destroy.js:92:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}
FetchError: request to https://sheets.googleapis.com/v4/spreadsheets/1vkw_Lku7F_F3F_iNmFFrDq9j7-tQ6EmZPOLpLt-s3TY/values/Aichi?key=AIzaSyDtI69z7dF2482nw6ZZbFaniKLDLMyFeIA failed, reason: connect ETIMEDOUT 172.217.15.74:443
    at ClientRequest.<anonymous> (/home/runner/work/covid19japan-data/covid19japan-data/node_modules/node-fetch/lib/index.js:1455:11)
    at ClientRequest.emit (events.js:315:20)
    at TLSSocket.socketErrorListener (_http_client.js:426:9)
    at TLSSocket.emit (events.js:315:20)
    at emitErrorNT (internal/streams/destroy.js:92:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}
Error: PatientListError: Unexpected empty patient list
    at Object.verifyPatientLists (/home/runner/work/covid19japan-data/covid19japan-data/src/verify.js:60:13)
    at Object.mergePatients (/home/runner/work/covid19japan-data/covid19japan-data/src/merge_patients.js:8:10)
    at /home/runner/work/covid19japan-data/covid19japan-data/generate.js:62:36
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
